packages/web/src/services/solana/WithdrawUSDC.ts

```diff
@@ -55,52 +55,43 @@ export const getSwapUSDCUserBankInstructions = async ({
     outputTokenSymbol: 'SOL',
     inputAmount: amount,
     slippage: USDC_SLIPPAGE,
-    swapMode: 'ExactOut' as SwapMode,
+    swapMode: 'ExactOut',
     onlyDirectRoutes: true
   })
-  const usdcNeededAmount = quoteRoute.inputAmount
-  const swapRoute = await JupiterSingleton.getQuote({
+  const usdcNeededAmount = quoteRoute.inputAmount.amount
+  const swapQuote = await JupiterSingleton.getQuote({
     inputTokenSymbol: 'USDC',
     outputTokenSymbol: 'SOL',
-    inputAmount: usdcNeededAmount.uiAmount,
+    inputAmount: Number(BigInt(usdcNeededAmount)),
```

Comment:
Why are we converting to a number after converting to a bigint?

Reply:
good catch